src/isEmpty.js

```diff
@@ -16,6 +16,8 @@ var _curry1 = require('./internal/_curry1');
  *      R.isEmpty([]); //=> true
  *      R.isEmpty(''); //=> true
  *      R.isEmpty(null); //=> false
+ *      R.isEmpty({}); //=> false
```

Comment:
I've updated the pull request. It now includes `R.isEmpty({length: 0})` as an example.